egs/chime4/s5_1ch/local/real_noisy_chime4_data_prep.sh

```diff
@@ -98,13 +102,17 @@ fi
 # data-preparation stage independent of the specific lexicon used.
 noiseword="<NOISE>";
 for x in $list_set;do
+  cat ${x}_wav.scptmp | awk '{print $1}' > $x.txtp1
+  cat $x.trans1 | awk '{$1=""; print $0}' | sed 's/^[ \t]*//g' > $x.txtp2
```

Comment:
best to clean up these temporary files when done; and please use better filenames.  Add ".tmp" instead of appending "temp", and what do txtp1 and txtp2 mean?  Either use better filenames or clarify in comments.